filament/include/filament/Camera.h

```diff
@@ -234,6 +249,19 @@ class UTILS_PUBLIC Camera : public FilamentAPI {
      */
     void setCustomProjection(math::mat4 const& projection, double near, double far) noexcept;
 
+    /** Sets a custom projection matrix for each eye.
+     *
+     * The projectionForCulling, near, and far parameters establish a "culling frustum" which must
+     * encompass anything either eye can see.
+     *
+     * @param projectionForCulling custom projection matrix for culling, must encompass both eyes
+     * @param near distance in world units from the camera to the culling near plane. \p near > 0.
+     * @param far distance in world units from the camera to the culling far plane. \p far > \p near.
+     * @see setCustomProjection
+     */
+    void setCustomEyeProjection(math::mat4 const (&projection)[2],
```

Comment:
Not a fan of the reference to an array, Instead I think it would be better to pass a `mat4 const*` and a count. You can even say that "count *must* be 2".

Reply:
I think if you prefer an "eye id" API, then we can do something similar to `setEyeModelMatrix`:

```
    void setCustomEyeProjection(uint8_t eyeId, math::mat4& projection),
```

Reply:
yes I'd like that, but this method also has parameters that are for head, so not related to a given eye. if we can resolve that it would be nice.

Reply:
Ah right, which is I think why I wrote it this way. I'll switch to `mat4 const*` for now.